src/components/DisplayAccounts.jsx

```diff
@@ -51,18 +52,23 @@ export default function DisplayAccounts({ accounts, ...props }) {
           </rb.Accordion.Header>
           <rb.Accordion.Body>
             <Link to="/send" state={{ account }} className="btn btn-outline-dark">
-              Send
+              {t('current_wallet_advanced.account_button_send')}
             </Link>{' '}
             <Link to="/receive" state={{ account }} className="btn btn-outline-dark">
-              Receive
+              {t('current_wallet_advanced.account_button_receive')}
             </Link>
             {branches.map(({ balance, branch, entries }) => {
               const [type, derivation, xpub] = branch.split('\t')
               return (
                 <article key={derivation}>
                   <rb.Row className="mt-4 pe-3">
                     <rb.Col>
-                      <h6>{titleize(type)}</h6>
```

Comment:
It seems function `titleize` is not needed anymore - should it be removed? Subsequently, functions `upcaseFirstChar`, `lowercaseFirstChar` and `dasherize` can also be removed from `utils.js`.